docs/update-cli.md

```diff
@@ -0,0 +1,19 @@
+# Update OpenAPI CLI
+
+If you've already installed OpenAPI CLI, you'll want to keep it as shiny and up to date as possible. Staying on top of versions also ensures that you get the latest functionality and bug fixes, as well as the most recent Redocly configuration file. A must-have!
```

Comment:
```suggestion
If you've already installed OpenAPI CLI, you'll want to keep it as up to date as possible. Staying on top of versions ensures that you get the latest functionalities and bug fixes.
```

Reply:
Suggesting this change because we're unnecessarily focusing on the config file. The current text somehow implies that the file is always changing, which is not true and may confuse the users.